LekceDveSamples/LekceDveSamples/UserLogin.feature

```diff
@@ -0,0 +1,10 @@
+﻿Feature: UserLogin
+	To show another possible naming options
+
+
+Scenario: Platny uzivatel se muze prihlasit
```

Comment:
~~Platny~~

Reply:
To bych nechal. Uz jen to ze ve stepech se mluvi o valid username a valid password. A pak kdyz mas tech testu vice tak to vypada tak ze mas 
Valid user
Iinvalid user
Valid user with wrong password

To platny tam dava smysl. Jen jsem to prepsal do anglictiny